Sources/CleevioUI/Views/Inputs/CleevioInputField.swift

```diff
@@ -101,41 +129,43 @@ public struct CleevioInputField<
     }
 
     var state: InputFieldState {
-        .init(
-            isFocused: isFocused || configuration.isExternallyFocused,
+        let isFocused = isFocused || configuration.isExternallyFocused
+        return .init(
+            isFocused: isFocused,
             isEnabled: isEnabled,
-            isError: error != nil
+            isError: configuration.errorPresentationCondition(isFocused, error)
         )
     }
 
+    var isErrorPresented: Bool {
+        state.isError
+    }
+
     public var body: some View {
         VStack(alignment: .leading, spacing: 6) {
```

Comment:
Maybe we can initialize the set just once to avoid multiple checks of the same?

`let state = self.state`